fair/RCPs/rcp3pd.py

```diff
@@ -0,0 +1,88 @@
+# Convenience module for loading in RCP emissions datasets
+#
+# Usage:
+#
+# import rcp3pd
+# rcp3pd.Emissions.co2
+
+import numpy as np
+import os
+emissions_filename = os.path.join(
+    os.path.dirname(__file__), 'RCP3PD_EMISSIONS.csv')
+forcing_filename = os.path.join(
+    os.path.dirname(__file__), 'RCP3PD_MIDYEAR_RADFORCING.csv')
+aviNOx_filename = os.path.join(
+    os.path.dirname(__file__), 'aviNOx_fraction.csv')
+fossilCH4_filename = os.path.join(
+    os.path.dirname(__file__), 'fossilCH4_fraction.csv')
+
+aviNOx_frac = np.loadtxt(aviNOx_filename, skiprows=5, usecols=1, delimiter=',')
```

Comment:
Agree this would be good and adds flexibility while reducing duplication

Reply:
I'm trying to think how I would implement this in a nice way. Currently, when running the model (assume I am not in the FAIR directory tree and FAIR is in $PYTHONPATH) the first few lines of my script will look like:

```
from fair.forward import fair_scm
from fair.RCPs import rcp85
from fair.ancil import cmip6_solar, cmip6_volcanic, natural
...
```

I'd like to keep this import structure, especially as you can do things like `emissions = rcp85.Emissions.co2`.

I'm not sure this would work if the Emissions and Forcing classes went into a separate file as I think you would have to write a function to return a class. You would end up with something like `emissions = rcp85.Emissions.Emissions.co2` if it worked at all (and `rcp85.Emissions.Emissions.emissions` for the full array!) 

I'm sure there's a way, but my brain doesn't permit one at the moment.

Reply:
There will definitely be a way to fiddle the imports so you can keep all your top level calls the same, as you say it will require a little bit of thinking. 

Once we make the whole thing object oriented it will be super easy to do so maybe it's not worth worrying about until then.